src/App.js

```diff
@@ -25,17 +25,23 @@ class App {
     let sum = 0;
     const inputArray = inputString.split("");
     inputArray.forEach((element) => {
+      //정수인 경우
       if (!!parseInt(element)) {
         currentValue += element;
       } else {
+        //currentValue가 비어있지 않은 경우
+        if (!parseInt(currentValue)) {
```

Comment:
```suggestion
        if (currentValue) {
```
이렇게 하지 않으신 이유가 있을까요? 주석에 적힌 것처럼 존재하는지만 판단하는 거라면 조건문에 `parseInt`가 왜 있는건지 궁금합니다!

Reply:
남겨주신 리뷰를 보고 한참 생각해보았는데, 처음 설계시 예외처리를 위해 생각해놓았던 조건인데 실제로 구현단계에서 막상 불필요해진 필터링인 것 같네요 ㅎㅎ 
전혀 생각지도 못했는데 지예님께서 꼼꼼하게 읽어주신 덕분에 캐치할 수 있었네요! 감사합니다!